ush/load_fv3gfs_modules.sh

```diff
@@ -13,16 +13,32 @@ ulimit_s=$( ulimit -S -s )
 source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ush/module-setup.sh"
 
-# Source versions file for runtime
-source "${HOMEgfs}/versions/run.ver"
-
 # Load our modules:
 module use "${HOMEgfs}/modulefiles"
 
 case "${MACHINE_ID}" in
-  "wcoss2" | "hera" | "orion" | "hercules" | "gaea" | "jet" | "s4" | "noaacloud")
+  "noaacloud")
+    #TODO this is a total kludge to get epic mount point for compute nodes
+    #    to be the same as the login node.  This should be workng from in the
+    #    ALLNODES section of the User Bootstrap of Parllel Works but it doen't
+    #    on the Rokcky Clusters (works fine in the Centos 7 cluster)
+    if [[ ! -d  /contrib-epic/EPIC ]]; then
+      if [[ -d  /contrib/Terry.McGuinness/SETUP ]]; then
+        /contrib/Terry.McGuinness/SETUP/mount-epic-contrib.sh
+        sudo systemctl daemon-reload
+      fi
+    fi
+    # Check if the OS is Rocky or CentOS
+    OS_NAME=$(grep -E '^ID=' /etc/os-release | sed -E 's/ID="?([^"]*)"?/\1/') || true
+    # Source versions file for runtime
+    source "${HOMEgfs}/versions/run.${MACHINE_ID}.${OS_NAME}.ver"
     module load "module_base.${MACHINE_ID}"
     ;;
+  "wcoss2" | "hera" | "orion" | "hercules" | "gaea" | "jet" | "s4")
+    # Source versions file for runtime
+    source "${HOMEgfs}/versions/run.${MACHINE_ID}.ver"
+    module load "module_base.${MACHINE_ID}"	    module load "module_base.${MACHINE_ID}"
+    ;;
```

Comment:
Couple comments for these lines:

1. You do not need the lines to source `run.${MACHINE_ID}.ver`. The lines above to source `run.ver` should come back. When the link script is run the appropriate `run.${MACHINE}.ver` is copied or linked to become `run.ver`.
2. The logic to decide which `run.${MACHINE_ID}.${OS_NAME}.ver` should be moved to `sorc/link_workflow.sh`. This way, the correct run version file becomes `run.ver`. As discussed in comment 1.
3. I don't think any changes are needed to `ush/load_fv3gfs_modules.sh`. The changes I discussed in comments 1 and 2 should move all of these changes to the appropriate place.

Let me know if you have any questions about my comments. :)